cockatrice/src/settingscache.h

```diff
@@ -176,7 +176,8 @@ class SettingsCache : public QObject {
     bool getRememberGameSettings() const { return rememberGameSettings; }
     int getKeepAlive() const { return keepalive; }
     void setClientID(QString clientID);
-    QString getClientID() { return clientID; }    
+    QString getClientID() { return clientID; }   
```

Comment:
I originally started out coding the unique server client id to be stored/called like the actual clientid value generated on startup so that it could be referenced later for who knows what ever reason down the road.  Later on while thinking about things it seems to me that the only time the function really should be called is prior to any remote client command which brought me to question 2 above.

If we want to go off that mentality the function should be moved into the remote client code base and made a private function.  Personally I like that idea so if you dont mind I will update it to that unless you have any thoughts on the matter?